workflows/workflows.go

```diff
@@ -0,0 +1,11 @@
+package workflows
+
+import "encoding/json"
+
+func ToWorkflowJson(item any) (string, error) {
+	if bytes, err := json.Marshal(item); err != nil {
+		return "", err
+	} else {
+		return string(bytes), nil
+	}
+}
```

Comment:
What is this used for? Also, can we move it to an `internal` directory as well?

Reply:
This should output JSON as the github actions expect it - no newlines, etc. It's Public so that a buildpack author can consume it.

Reply:
Sorry, but can you add a comment for it describing that?

Reply:
along with a  line with how where users might expect to use it

Reply:
comment to be pushed with next commit